modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/common/components/PRMForm/components/fields/InputFile/InputFile.tsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import {ClayInput} from '@clayui/form';
+
+import setFileFormik from '../../../../../utils/setFileFormik';
+import WrapperInput from '../common/components/WrapperInput';
+import PRMFormFieldProps from '../common/interfaces/prmFormFieldProps';
+import PRMFormFieldStateProps from '../common/interfaces/prmFormFieldStateProps';
+
+interface IProps {
+	activityId?: number;
+	budgetId?: number;
+	mdfRequestId?: number;
+	setFieldValue: (
+		field: string,
+		value: any,
+		shouldValidate?: boolean | undefined
+	) => void;
+	typeDocument: string;
+}
+
+const InputFile = ({
+	activityId,
+	budgetId,
+	field,
+	label,
+	mdfRequestId,
+	meta,
+	required,
+	setFieldValue,
+	typeDocument,
+	...props
+}: PRMFormFieldProps &
+	PRMFormFieldStateProps<string> &
+	React.ComponentProps<typeof ClayInput> &
+	IProps) => {
+	return (
+		<WrapperInput {...meta} label={label} required={required}>
+			<ClayInput
+				{...props}
+				onChange={(event) => {
+					setFileFormik(
+						event.target.files,
+						field.name,
+						activityId,
+						budgetId,
+						mdfRequestId,
+						setFieldValue,
+						typeDocument
+					);
```

Comment:
This function could receive an object instead of those 7 parameters? it's hard to identify each field and their responsibility 

```suggestion
					setFileFormik(
						event.target.files,
						field.name,
						activityId,
						budgetId,
						mdfRequestId,
						setFieldValue,
						typeDocument
					);
```